ElliotKillick/Mido
# chore: update win11x64-enterprise-eval sha256sum

### sha256sum

win11x64-enterprise-eval sha256sum = c8dbc96b61d04c8b01faf6ce0794fdf33965c7b350eaa3eb1e6697019902945c

### Bing Search

https://www.bing.com/search?q=%22c8dbc96b61d04c8b01faf6ce0794fdf33965c7b350eaa3eb1e6697019902945c%22

- https://github.com/rgl/windows-vagrant/blob/8d66fe7638d2815901574bfd011840a65c7f49a1/windows-11-23h2-uefi.pkr.hcl#L33
- https://files.rg-adguard.net/file/b6a83acc-7bc3-b46e-675d-4070987243c7

### Logs

![image](https://github.com/ElliotKillick/Mido/assets/29926675/f22d8f8c-d8cb-45d5-ad8e-e00ee950b001)


```
$ Mido.sh win11x64-enterprise-eval                                                                                  ─╯
[i] Downloading Windows media from official Microsoft servers...
[i] Downloading Windows 11 Enterprise Evaluation...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  120k  100  120k    0     0   152k      0 --:--:-- --:--:-- --:--:--  152k
[+] Got latest ISO download link: https://software-static.download.prss.microsoft.com/dbazure/888969d5-f34g-4e03-ac9d-1f9786c66749/22631.2428.231001-0608.23H2_NI_RELEASE_SVC_REFRESH_CLIENTENTERPRISEEVAL_OEMRET_x64FRE_en-us.iso
################################################################################################################# 100.0%
[i] Verifying integrity...
win11x64-enterprise-eval.iso: UNVERIFIED

[i] Manual verification instructions
    1. Get checksum (may already be done for you):
    sha256sum <ISO_FILENAME>

    2. Verify media:
    Web search: https://duckduckgo.com/?q=%22CHECKSUM_HERE%22
    Onion search: https://duckduckgogg42xjoc72x3sjasowoarfbgcmvfimaftt6twagswzczad.onion/?q=%22CHECKSUM_HERE%22
    "No results found" or unexpected results indicates the media has been modified and should not be used.

    3. Remove the .UNVERIFIED extension from the file after performing or deciding to skip verification (not recommended):
    mv <ISO_FILENAME>.UNVERIFIED <ISO_FILENAME>

    win11x64-enterprise-eval.iso.UNVERIFIED = c8dbc96b61d04c8b01faf6ce0794fdf33965c7b350eaa3eb1e6697019902945c
        Web search: https://duckduckgo.com/?q=%22c8dbc96b61d04c8b01faf6ce0794fdf33965c7b350eaa3eb1e6697019902945c%22
        Onion search: https://duckduckgogg42xjoc72x3sjasowoarfbgcmvfimaftt6twagswzczad.onion/?q=%22c8dbc96b61d04c8b01faf6ce0794fdf33965c7b350eaa3eb1e6697019902945c%22
        mv win11x64-enterprise-eval.iso.UNVERIFIED win11x64-enterprise-eval.iso

    Theses searches can be performed in a web/Tor browser or more securely using
    ddgr (Debian/Fedora packages available) terminal search tool if preferred.
    Once validated, consider updating the checksums in Mido by submitting a pull request on GitHub.
[!] 1 of the downloaded Windows media did NOT match the expected checksum! This means either that the media is a newer release than our current checksum (stored in Mido), was corrupted during download, or that is has been (potentially maliciously) modified! Please manually verify the Windows media before use: win11x64-enterprise-eval.iso
[+] Finished! Please see the above errors with information
```
